### DISCUSSION TITLE
latest version don't have arm-v7

### DISCUSSION THREAD

**Discussion by zeev-mindali** [zwave-js/zwave-js-ui#4016] [General]
Dear all,,
is there a reson why the new releases does not includ the arm-v7 support?

zeev

**Comment by robertsLando** [comment]
Just submitted a possible fix

**Reply by robertsLando** [reply]
Check https://github.com/zwave-js/zwave-js-ui/actions/runs/11897038866

Here is the binary: https://github.com/zwave-js/zwave-js-ui/actions/runs/11897038866/artifacts/2202709044
